c/include/nnstreamer.h

```diff
@@ -141,6 +141,7 @@ typedef enum {
   ML_NNFW_TYPE_PYTORCH = 11,          /**< PyTorch (.pt). (Since 6.5) */
   ML_NNFW_TYPE_NNTR_INF = 12,         /**< Inference supported from NNTrainer, SR On-device Training Framework (Since 6.5) */
   ML_NNFW_TYPE_SNAP = 0x2001,         /**< SNAP (Samsung Neural Acceleration Platform), only for Android. (Since 6.0) */
+  ML_NNFW_TYPE_SRNPU = 0x3000,        /**< SR (Samsung Research) made NPU (.tvn). (Since 6.5) */
```

Comment:
plz set ```ML_NNFW_TYPE_SRNPU == 13```

Reply:
Updated, thanks.

Reply:
Oh..I didnt check this PR, can you rebase and udpate with new enum value ML_NNFW_TYPE_SRNPU = 14